server/bin/pulp-qpid-ssl-cfg

```diff
@@ -14,7 +14,11 @@
 #
 
 DIR="/tmp/tmp$RANDOM"
-HOST=`hostname`
+HOST=`hostname -f`
+if [ -z "$HOST" ]
```

Comment:
It's pretty common in my experience for the fqdn to not resolve properly locally, resulting in `hostname -f` outputting an error to STDERR and nothing to STDOUT. In this case, there are two situations I can think of:

- Non-fqdn hostname contains the fqdn
- Non-fqdn hostname contains just a shortname
- Hostname is completely misconfigured.

This attempts to handle the possibilities as gracefully as possible by preferring the DNS-resolved FQDN, and falling back to what the system thinks its hostname is, which may actually be resolvable within the same domain/search suffix.